### PR TITLE
Fix typo

### PR DESCRIPTION
**Pull request by fukatani**

### CODE REVIEW

File: chainer/links/connection/dilated_convolution_2d.py [hunk 1]
```diff
@@ -31,11 +31,11 @@ class DilatedConvolution2D(link.Link):
         nobias (bool): If ``True``, then this link does not use the bias term.
         use_cudnn (bool): If ``True``, then this link uses cuDNN if available.
         initialW (4-D array): Initial weight value. If ``None``, then this
-            function uses to initialize ``wscale``.
+            function uses scaled Gaussian ditribution to initialize weight.
```
Comment (mitmul) [inline chainer/links/connection/dilated_convolution_2d.py]: `ditribution` should be a typo for `distribution`

File: chainer/links/connection/highway.py [hunk 2]
```diff
@@ -28,16 +28,18 @@ class Highway(link.Chain):
         nobias (bool): If ``True``, then this function does not use the bias.
         activate: Activation function of plain array. :math:`tanh` is also
             available.
-        init_Wh (2-D array): Initial weight value of plain array. If ``None``,
-            then this function uses it to initialize ``wscale``.
+        init_Wh (2-D array): Initial weight value of plain array.
+            If ``None``, then this function uses Gaussian distribution scaled
```
Comment (mitmul) [inline chainer/links/connection/highway.py]: Is this line break right before `If None` necessary?

File: chainer/links/connection/linear.py [hunk 3]
```diff
@@ -29,11 +29,12 @@ class Linear(link.Link):
         bias (float): Initial bias value.
         nobias (bool): If ``True``, then this function does not use the bias.
         initialW (2-D array): Initial weight value. If ``None``, then this
-            function uses to initialize ``wscale``.
+            function uses Gaussian ditribution scaled by ``w_scale`` to
```
Comment (mitmul) [inline chainer/links/connection/linear.py]: `ditribution` should be a typo for `distribution`.

**Comment by fukatani** [comment]
Thanks for your comments!
It's embarrassing :smiley: But I fixed them.

**Comment by mitmul** [comment]
Thank you for quick revising. LGTM

**Pull request merged**